internal/stacks/stack.go

```diff
@@ -422,15 +436,15 @@ func (s *Stack) patchConfigAndRestartFireflyNodes(verbose bool) error {
 	return nil
 }
 
-func (s *Stack) restartFireflyNodes(verbose bool) error {
-	for _, member := range s.Members {
-		containerName := fmt.Sprintf("%s_firefly_core_%s_1", s.Name, member.ID)
-		if err := s.restartFireflyNode(containerName, verbose); err != nil {
-			return err
-		}
-	}
-	return nil
-}
+// func (s *Stack) restartFireflyNodes(verbose bool) error {
+// 	for _, member := range s.Members {
+// 		containerName := fmt.Sprintf("%s_firefly_core_%s_1", s.Name, member.ID)
+// 		if err := s.restartFireflyNode(containerName, verbose); err != nil {
+// 			return err
+// 		}
+// 	}
+// 	return nil
+// }
```

Comment:
If we don't need this code anymore, should we just get rid of it? If we need to retrieve it again, it's in git history.

Reply:
sorry @nguyer - I'm actually not sure if we do or not, but the warnings in VSCode were it isn't used.
If you think it's good to go, I'll remove it 👍

Reply:
No worries. If it's unused, I vote: remove. Dead code is is tech debt.